openslides_backend/http/application.py

```diff
@@ -63,10 +63,21 @@ def dispatch_request(self, request: Request) -> Union[Response, HTTPException]:
                 raise
         except HTTPException as exception:
             return exception
+        if type(response_body) == dict:
+            status_code = response_body.get("status_code", 200)
+        elif request.path == r"/system/presenter/handle_request":
```

Comment:
Mmh, I don't like that the generic application should have specific knowledge of the underlying views. Can you find a more generic way to handle this here? What is even the problem?

Reply:
Talked with Raimund: I don't know a more generic way. First case should be the standard: Response with status_code in a dict for actions, presenter or anything else, that may be.
2nd case: Presenter without status code: Finally a kind of error, but in practice working, consequence: Let it pass and assume a 200.
2rd case: An action or anything else without status code: throw an exception, this is not allowed, because potentially dangerous.
I don't have a better solution!